### PR TITLE
add @transloadit/trigger-pager

### PR DESCRIPTION
**Pull request by juliangruber**

### CODE REVIEW

File: packages/trigger-pager/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+await triggerPager({
+  token,
+  serviceId,
+  incidentKey: 'OHNO',
+  description: 'the bad thing',
+  // urgency: 'high',
+  // title: 'OHNO',
+})
+```
```
Comment (juliangruber) [inline packages/trigger-pager/README.md:18]: Individual services can export their own wrappers, with defaults set, eg

```js
// lib/trigger-pager.js

const triggerPager = require('@transloadit/trigger-pager')

module.exports = args => triggerPager({
  token: process.env.TOKEN,
  serviceId: process.env.SERVICE_ID,
  ...args
})
```

**Review by tim-kos — APPROVED** [review]
Cool!

File: packages/trigger-pager/triggerPager.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  const res = await api({ token }).post('/incidents', {
+    headers: {
+      from,
+    },
+    data: {
+      incident: {
+        type        : 'incident',
+        incident_key: incidentKey,
+        urgency,
+        title       : title.length >= 1024
```
Comment (kvz) [inline packages/trigger-pager/triggerPager.js:24]: You could use abbr, also in this repo. It abbreviated in the middle, which often can often be helpful-ish as identifiers or counters often come last

**Review by kvz — APPROVED** [review]
Very nice! Just one tip but feel free to ignore just as well.

File: packages/trigger-pager/triggerPager.js (hunk 2 above)
Comment (juliangruber) [reply packages/trigger-pager/triggerPager.js:24]: Good tip! Let's add that when it becomes relevant, I just wanna get this out 😎

**Pull request merged**